package.json

```diff
@@ -19,6 +19,7 @@
     "@voiceflow/body-parser": "^1.21.0",
     "@voiceflow/common": "6.5.0",
     "@voiceflow/general-runtime": "1.56.1",
+    "@voiceflow/internal": "1.7.0",
```

Comment:
unfortunately `@voiceflow/internal` is a private package and this is a public repo/package, so we'll need to use a different package for the source of the `PlatformType` enum